metricbeat/module/kafka/README.md

```diff
@@ -0,0 +1,101 @@
+### Manually testing Kafka modules
+
+Testing Kafka can be tricky, so the purpose of this doc is to document all the steps that one should follow in order to
+prepare an environment and manually test Kafka module.
+
+#### Kafka container
+
+In order to have a Kafka instance up and running the best way to go is to use the container that is used by the CI tests.
+To bring this container up simply run the tests for Kafka module:
+
+`go test -tags integration ./metricbeat/module/kafka/...`
+
+
+After the tests have been completed, the Kafka container should be still running. Verify with:
+
+```console
+707b50334835    docker.elastic.co/integrations-ci/beats-kafka:2.1.1-2  "/run.sh"        2 minutes ago    Up 2 minutes (healthy)  2181/tcp, 0.0.0.0:32785->8774/tcp, 0.0.0.0:32784->8775/tcp, 0.0.0.0:32783->8779/tcp, 0.0.0.0:32782->9092/tcp  kafka_a035cf4c6889705a_kafka_1
+```
+
+In order to identify to which port the Broker is listening on one should check in the logs of the container and find 
+the advertised address:
+
+```console
+docker logs 707b50334835 > kafka_logs
+cat kafka_logs | grep OUTSIDE
+
+advertised.listeners = INSIDE://localhost:9091,OUTSIDE://localhost:32778
+listener.security.protocol.map = INSIDE:SASL_PLAINTEXT,OUTSIDE:SASL_PLAINTEXT
+listeners = INSIDE://localhost:9091,OUTSIDE://0.0.0.0:9092
+advertised.listeners = INSIDE://localhost:9091,OUTSIDE://localhost:32778
+listener.security.protocol.map = INSIDE:SASL_PLAINTEXT,OUTSIDE:SASL_PLAINTEXT
+listeners = INSIDE://localhost:9091,OUTSIDE://0.0.0.0:9092
+```
+
+So here in this example the host we should in the module's config is `localhost:32778`.
+Note that this is different between MAC and Linux machines. The above is the case for the MAC machine, and here is how 
+the respective address for a LINUX machine should look like:
+
+```console
+advertised.listeners = INSIDE://localhost:9091,OUTSIDE://172.26.0.2:9092
+listener.security.protocol.map = INSIDE:SASL_PLAINTEXT,OUTSIDE:SASL_PLAINTEXT
+listeners = INSIDE://localhost:9091,OUTSIDE://0.0.0.0:9092
+advertised.listeners = INSIDE://localhost:9091,OUTSIDE://172.26.0.2:9092
+listener.security.protocol.map = INSIDE:SASL_PLAINTEXT,OUTSIDE:SASL_PLAINTEXT
+listeners = INSIDE://localhost:9091,OUTSIDE://0.0.0.0:9092
+```
+
+So here the advertised addressed to be used in the config is `172.26.0.2:9092`.
+
+This difference comes from here: https://github.com/elastic/beats/blob/master/libbeat/tests/compose/wrapper.go#L137
```

Comment:
Please use a fixed version in this URL in case this code is moved.
```suggestion
This difference comes from here: https://github.com/elastic/beats/blob/v7.5.2/libbeat/tests/compose/wrapper.go#L137
```